common/README.md

```diff
@@ -0,0 +1,69 @@
+# Common Definitions
+
+This folder contains common definitions for protobufs used more than once. We intentionally have this in order to standardize the way we define the data to make it easier for the end user to "plug-n-play".
+
+## Common Proto
+
+TODO
+
+## DEX AMM Proto
+
+TODO
+
+## Token Proto
+
+The token proto is designed to store the token balances of all accounts throughout the blockchains history. It is designed to be network-agnostic and support all tokens. Token's can be blockchain native, erc20s, or any other derivative.
```

Comment:
What about NFTs? Would they just adopt the `Token` definition as well or would they have a separate one?

Reply:
I should fix the verbiage, this was meant to be only for fungible tokens. And it could fit the erc20, maybe not, and to be inclusive to other blockchain standards like solana SPL tokens.

I think it is best to make a separate proto def for NFTs, I think from our experience it is best to make schemas focused. That approach also feels much more native to substreams. Would you agree?

Reply:
lmk what you think of the proto named evm.token.proto? Do you think it should like in that folder or should I make the path `./common/proto/token/evm.token.proto` assuming we will have more

Reply:
Why `evm.token` instead of `erc20`? ERC20 is more of a convention and I haven't seen anyone used the term EVM token. This can cause confusion.

Reply:
Yeah I guess only ETH, I will change to erc20